.gitlab-ci.yml

```diff
@@ -0,0 +1,58 @@
+---
+
+image: govcms/govcms-ci:edge2
```

Comment:
https://github.com/govCMS/govcms-ci/pull/5

Reply:
This can change back to a stable release, has been merged